web/docs/auth/overview.md

```diff
@@ -295,7 +296,7 @@ Since the `user` prop is only available in a page's React component: use the `us
 
 #### Using the `context.user` object
 
-When authentication is enabled, all [queries and actions](/docs/data-model/operations/overview) have access to the `user` object through the `context` argument. `context.user` contains all User entity's fields, except for the password.
+When authentication is enabled, all [queries and actions](/docs/data-model/operations/overview) have access to the `user` object through the `context` argument. `context.user` contains all User entity's fields and the auth identities connected to the user. We strip out the `hashedPassword` field from the identities for security reasons.
```

Comment:
This idea that we have `user.auth.identities` -> we should make that very clear and obvious in docs. I am not saying we need to say it first, it is advanced stuff, but when the moment for advanced stuff comes (for example here in this overview), then I would take some space and their focus to introduce just this specific concept very clearly.

Maybe a very simple diagram, or maybe `user.auth.identities` code block in the middle of the page, with very clear explanation -> your user object now has auth field, added by Wasp. That field has `identities` field, which is an array contains info about all the auth methods (username / pass, email / pass, github, google, ...) that your user has signed up with so far. Via this array you can get detailed info about each of the identities -> which email they use for email / pass, what is their github id, ... .

And maybe some more diagrams or something.

Because this is a bit weird and out of blue, and is really the most important thing to remember about the whole Auth / AuthIdentity business -> if they remember this, they will know how to look up the rest.

Reply:
I've added an intro with a code block here 👍